### DISCUSSION TITLE
Coding guidelines

### DISCUSSION THREAD

**Discussion by tm1000** [stagehacks/Cue-View#265] [General]
Just curious for those of us who want to contribute. 

- Can we use ESM import/export
- Thoughts on web workers?
- I've been reviewing electron performance guidelines and I see that using a bundler could improve performance
- really need to have a license file in here otherwise everything is copyrighted to @sparks-alec which is fine if that's what they want but it would be good to know

**Comment by tm1000** [comment]
@sparks-alec

**Comment by tm1000** [comment]
@sparks-alec do you have discord? @jwetzell And I have met up there

**Comment by sparks-alec** [comment]
I'm gonna be honest and say I did not realize the Discussions area had anything in it because there was no number next to it. Apologies.🤦🏻‍♂️

Happy to chat on Discord- let me know where to go. Open to starting a Cue View server too.

**Reply by jwetzell** [reply]
Maybe a generic "Stage Hacks" server?

**Reply by tm1000** [reply]
I'm down for a Stage Hacks server as I could see myself contribing on a lot of things. Particularly if they become rust projects ;-)
